src/test/SmartComponents/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -338,5 +337,57 @@ describe('Click through all steps', () => {
         await screen.findByText('rhel-8');
         await screen.findByText('aws');
         await screen.findByText('Register the system on first boot');
+
+        // mock the backend API
+        const composeImage = jest.spyOn(api, 'composeImage');
+        composeImage.mockImplementation(async () => ({ id: 'test-me' }));
+
+        const create = screen.getByRole('button', { name: /Create/ });
+        create.click();
+
+        // API request sent to backend
+        expect(composeImage).toHaveBeenCalled();
+
+        // todo: assertion fails, can't find the button and we don't know why !
+        // returns back to the landing page
+        // await waitFor(
+        //    () => [
+        //        screen.getByText('Create a new image'),
```

Comment:
Hm I think this should be replaced here (and in verifyCancelButton) with something that is exclusive to the landing page, as the text "Create a new image" occurs on both the landing page and the wizard.

How about `screen.getByTestId('create-image-action')`, `images-table` as testId. Feel free to add a new one to on the landing page as well.